src/main/java/bio/terra/app/controller/UnauthenticatedApiController.java

```diff
@@ -25,6 +30,12 @@
 
     private final OauthConfiguration oauthConfig;
 
+    private Logger logger = LoggerFactory.getLogger(UnauthenticatedApiController.class);
+
+    private static String semVer;
```

Comment:
Sorry, I wasn't super clear when we spoke.  What I was thinking is that you could have static vars for the defaults and these member variables should be `private final`.